bin/docker-image-tool.sh

```diff
@@ -145,6 +157,8 @@ Options:
   -f file               Dockerfile to build for JVM based Jobs. By default builds the Dockerfile shipped with Spark.
   -p file               Dockerfile to build for PySpark Jobs. Builds Python dependencies and ships with Spark.
   -R file               Dockerfile to build for SparkR Jobs. Builds R dependencies and ships with Spark.
+  -pskip                Skip building PySpark docker image.
```

Comment:
I see what you did here.

But it would be better to document this as part of the "-p" and "-R" switches.

Reply:
Done. Updated the usage to document 'skip' as part of -p and -R switches.